pom.xml

```diff
@@ -2332,6 +2332,20 @@
             </properties>
         </profile>
 
+        <profile>
+            <id>fast</id>
+            <properties>
+                <checkstyle.skip>true</checkstyle.skip>
+                <maven.javadoc.skip>true</maven.javadoc.skip>
+                <maven.resources.skip>true</maven.resources.skip>
+                <rat.skip>true</rat.skip>
+                <skipTests>true</skipTests>
+                <spotless.check.skip>true</spotless.check.skip>
+                <!-- skip generating scaladoc-->
+                <skip>true</skip>
```

Comment:
Can we give it a meaningful name? maybe merge w/ maven.javadoc.skip into one.

Reply:
The scala-maven-plugin says it's using property `skip` in the goal `scala:doc-jar`.  <https://davidb.github.io/scala-maven-plugin/doc-jar-mojo.html#skip>

Or let me change to set this with `pluginManagement` within profile instead of using property setting here.